llvm-general/test/LLVM/General/Test/Metadata.hs

```diff
@@ -73,6 +73,7 @@ tests = testGroup "Metadata" [
           MetadataNodeDefinition (MetadataNodeID 0) [ Just $ MDValue $ ConstantOperand (C.Int 32 1) ]
          ]
     let s = "; ModuleID = '<string>'\n\
+            \source_filename = \"<string>\"\n\
```

Comment:
Hm I thought I added it, but apparently I lost the commit somewhere. Let’s move this to a separate PR.